code/datums/components/food/edible.dm

```diff
@@ -331,8 +332,8 @@ Behavior that's still missing from this component that original food items had t
 	if(eater.satiety > -200)
 		eater.satiety -= junkiness
 	playsound(eater.loc,'sound/items/eatfood.ogg', rand(10,50), TRUE)
+	SEND_SIGNAL(parent, COMSIG_FOOD_EATEN, eater, feeder, bitecount, bite_consumption)
```

Comment:
This shouldn't have been moved. the check exists for a reason.

Reply:
I have not found any issue with the move, but i accept this can be important for something not seen.
Would you rather I add a first bite call back or signal?

Reply:
I would rather you did neither as this fix is bad either way. Adding reagents on first bite is bad practice.